README.md

```diff
@@ -66,6 +65,14 @@ After running `nix develop` you should automatically have a number of support sc
 - [`format`](scripts/format) formats all files. The format is enforced by our CI, so you should run this script prior to committing.
 - [`tests`](scripts/tests) run functional, kat tests natively or emulate them using `QEMU`. For information on how to use the script, please refer to the `--help` option.
 
+## Formal Verification
+
+We are currently aiming to prove that the C code is memory-safe, type-safe, and free of undefined behaviour, using the [CBMC](https://github.com/diffblue/cbmc) verification tool.
+The nix environment automatically installs CBMC and related tools.
```

Comment:
```
... when using the `ci-cbmc` shell.
nix develop --experimental-features 'nix-command flakes' .#ci-linter
```

Reply:
I just do 
```
nix develop --extra-experimental-features flakes --extra-experimental-features nix-command
```
from the root directory of the repo, and everything works... is that not right?